sbt-plugin-test/build.sbt

```diff
@@ -138,8 +138,13 @@ lazy val multiTest = crossProject.
   jsSettings(baseSettings: _*).
   jsSettings(
     name := "Multi test framework test JS",
+
     // Make FrameworkDetector resilient to other output - #1572
-    jsDependencies in Test += ProvidedJS / "consoleWriter.js",
+    jsExecutionFiles in Test := {
+      val consoleWriter = FileVirtualJSFile(
+          baseDirectory.value / "src/test/resources/consoleWriter.js")
```

Comment:
Piece of self-review: use `(resourceDirectory in Test).value / "consoleWriter.js"` instead.